go.mod

```diff
@@ -2,6 +2,7 @@ module go.thethings.network/lorawan-stack/v3
 
 go 1.17
 
+replace github.com/TheThingsIndustries/protoc-gen-go-flags => ../../github.com/TheThingsIndustries/protoc-gen-go-flags 
```

Comment:
Is this still needed ? Isn't the latest version tagged ?

Reply:
yep, I am using this to make ad-hoc fixes and accidentally committed it.